app-frontend/src/app/pages/browse/browse.component.js

```diff
@@ -0,0 +1,8 @@
+import browserTemplate from './browse.html';
```

Comment:
I've gone ahead and reverted these changes for now, and opened #637 to deal with refactoring controllers generally.